Formula/iproute2mac.rb

```diff
@@ -1,11 +1,12 @@
 class Iproute2mac < Formula
   desc "CLI wrapper for basic network utilities on macOS - ip command"
   homepage "https://github.com/brona/iproute2mac"
-  url "https://github.com/brona/iproute2mac/releases/download/v1.2.3/iproute2mac-1.2.3.tar.gz"
-  sha256 "95ef8d4b0e32e4d3e3d975afa11d7aa0797d59b0f2c21b73a4e26d357bd6f93f"
+  url "https://github.com/brona/iproute2mac/releases/download/v1.3.0/iproute2mac-1.3.0.tar.gz"
+  sha256 "3fefce6b0f5e166355fdb04934cbdd906211b64e5adb6a385469696dc51233b7"
   license "MIT"
 
   bottle :unneeded
+  depends_on "python@3.8"
```

Comment:
can you attach some reference for the `python 3.8`? I wonder if we can just use the os default python.

Reply:
I added the Python 3 dependency as on older Mac OS (Before Catalina) has only Python 2. When I used `depends_on: "python"`, I got `Dependency 'python' from homebrew/core is an alias; use the canonical name 'python@3.8'` from the audit test, so I switched that to 3.8.

The script uses `#!/usr/bin/env python3`, what is your recommendation to make this work on older systems as well?

Reply:
We should use the python 3 dependency. Or `uses_from_macos "python" :since => :catalina`

Reply:
Done